src/lib/viewers/BaseViewer.js

```diff
@@ -1100,6 +1101,18 @@ class BaseViewer extends EventEmitter {
         this.emit('annotatorevent', data);
     }
 
+    handleAnnotationCreateEvent({ annotation, meta: { status } = {} }) {
+        if (status !== 'pending') {
+            const activeMode = this.annotationControls.getActiveMode();
+            this.annotator.toggleAnnotationMode(activeMode);
+            this.annotationControls.resetControls();
+        }
+
+        if (status === 'success') {
+            this.annotator.emit('annotations_active_set', annotation.id);
```

Comment:
Is it possible that the user may click on another annotation during the save XHR? Do we want to try to respect that or is it an edge case we're not worried about right now?

Reply:
I guess it's possible? I think for now we can leave it, I'll follow up with design, my guess is we may want to disable interactivity during the `isPending` state?

Reply:
Probably just leave it as-is for now after validating with design.